src/node_sea.cc

```diff
@@ -73,6 +80,99 @@ std::tuple<int, char**> FixupArgsForSEA(int argc, char** argv) {
   return {argc, argv};
 }
 
+struct SeaConfig {
```

Comment:
I don't think we generally require them to be in header files to be in these sub namespaces? Similarly almost everything is in `node::` and we don't move them out to anonymous namespaces.

Reply:
I meant that since these are not declared in the header file, it's not meant to be used outside this translation unit, so these are supposed to have static linkage. Putting this in an anonymous namespace in this file is just a way to mark these as static.

Reply:
I don't think we generally care too much about static linkage unless there is a risk of conflicts (e.g. the name is very generic). But yeah I can move it to an anonymous namespace.